velox/dwio/parquet/writer/Writer.h

```diff
@@ -196,6 +197,10 @@ class Writer : public dwio::common::Writer {
   // Sets the memory reclaimers for all the memory pools used by this writer.
   void setMemoryReclaimers();
 
+  // Collect Iceberg data file statistics from FileMetaData.
+  void collectDataFileStats(
```

Comment:
collectDataFileStats -> collectIcebergDataFileStats

Reply:
The statistic is tightly related to iceberg, the key is iceberg field id, so we cannot use a generic name for it.